docs/installation_linux.rst

```diff
@@ -89,7 +89,13 @@ Clone code
    cd lisa
 
 
-Directly install LISA (Option 1)
+Install in Dev environment (Option 1)
+-------------------------------------
+
+For making any code changes to LISA, you will need to setup a development environment. Please follow the instructions in :ref:`DevVirtEnv`.
+
+
+Directly install LISA (Option 2)
```

Comment:
How about something like:
1. Development environment
2. Development environment without `nox`
3. Runtime environment

Reply:
> How about something like:
> 
> 1. Development environment
> 2. Development environment without `nox`
> 3. Runtime environment

It should be **scenario** based, instead of describing the underlying ways. User doesn't need to learn what's "nox", or when the "runtime" environment is needed. We can provide two options, development or run cases. Give an introduction.  explain what "development" means (like change lisa code).

Reply:
Sure! Let me update